src/components/Timer/Timer.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from "./Timer.module.scss";
+import { FC } from "react";
+
+const Timer: FC = () => {
+
+  return (
+    <div className={styles.timerContainer}>
+      <div className={styles.container}>
+        <div className={styles.progress} style={{"--i":85}}>
```

Comment:
`style={{"--i":85}` ?

Reply:
I'm passing i as value that indicates how many percent of time has passed. I will change it dynamically in the future. I change the name to percentOfTimePassed.